charters/threat-modeling-cg.html

```diff
@@ -0,0 +1,276 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>
+      Threat Modeling Community Group Charter
+    </title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="//www.w3.org/StyleSheets/TR/base">
+    <style>
+      body {
+        max-width: 60em;
+        margin: auto;
+      }
+      *:target {
+        background-color: yellow;
+      }
+      li {
+        margin-bottom: 9pt;
+      }
+      .note {
+        background-color: yellow;
+        padding: 10px;
+      }
+      .remove {
+        background-color: yellow;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>
+      [DRAFT] Threat Modeling Community Group Charter
+    </h1>
+    <p>
+      <span class="remove">{TBD: remove next sentence before submitting for
+      approval}</span> This Charter is work in progress. To submit feedback,
+      please use <a href="https://github.com/w3c-cg/threat-modeling/issues?q=is%3Aissue+is%3Aopen+%5Bcharter%5D">GitHub repository Issues</a>
+      where Charter is being developed.
+    </p>
+    <ul>
+      <li>This Charter: <a href="https://w3c-cg.github.io/threat-modeling/charter/threat-modeling-cg.html">https://w3c-cg.github.io/threat-modeling/charter/threat-modeling-cg.html</a>
+      </li>
+      <li>Previous Charter: <a href="https://w3c-cg.github.io/threat-modeling/charter/threat-modeling-cg.html">https://w3c-cg.github.io/threat-modeling/charter/threat-modeling-cg.html</a>
+      </li>
+      <li>Start Date: <span class="remove">{TBD: date the charter takes effect,
+      estimate if not known. Update this if the charter is revised and include
+      a link to the previous version of the charter.}</span>
+      </li>
+      <li>Last Modified: <span class="remove">{TBD: If the system does not
+      automatically provide information about the date of the last
+      modification, it can be useful to include that in the charter.}</span>
+      </li>
+    </ul>
+    <h2 id="goals">
+      Goals
+    </h2>
+    <p>
+      The group aims to provide a meeting point for Security, Privacy, Human Rights experts, and technology domain experts to create Threat Models together, following the <a href="https://www.threatmodelingmanifesto.org">Threat Modeling Manifesto</a>.
+      <br/>
+      This group will not publish Specifications.
+    </p>
+    <h2 id="scope-of-work">
+      Scope of Work
+    </h2>
+    <p>
+      Create Threat Models and other groups that develop specifications or otherwise produce other deliverables, providing specific expertise related to 
+      Threat Modeling and threat categories such as Security, Privacy, and Human Rights. 
```

Comment:
This is not quite a sentence. I cannot detect sufficient intended meaning to offer a suggested revision.

Reply:
Hi @TallTed thank you I'm preparing a separate PR